setup.py

```diff
@@ -22,6 +22,7 @@
             "st_referencemaps=shimmingtoolbox.cli.referencemaps:main",
             "st_b0maps=shimmingtoolbox.cli.b0map:main",
             "st_download_data=shimmingtoolbox.cli.download_data:download_data",
+            "st_mask_shimmingtoolbox=shimmingtoolbox.cli.mask_shimmingtoolbox:main",
```

Comment:
```suggestion
            "st_mask=shimmingtoolbox.cli.mask:main",
```

The name of the file should change to fit the new name.